### DISCUSSION TITLE
import different component to part of layout in different pages

### DISCUSSION THREAD

**Discussion by farshadfahimi** [nuxt/nuxt#7942] [General]
right now we have layout like below

```javascript
<template>
  <div id="app">
    <div>
      <div class="bg-dark d-block">
        <div class="default">
        </div>
        <div class="breadcrumb"></div>
      </div>

      <nuxt />
    </div>
  </div>
</template>
```

and i need to fill `class='breadcrumb'` content in pages

Is it possible or not?

**Comment by gianpesto** [comment]
One solution could be to use Vuex. You would populate data in a page and then use it in the layout.

**Comment by TheAlexLichter** [comment]
It is possible by using "named views" but unfortunately there isn't a 100% baked in version.

Further read:
https://nuxtjs.org/guide/routing#named-views
https://nuxtjs.org/examples/named-views
https://github.com/nuxt/nuxt.js/pull/4410
